test/kubemark/gce/util.sh

```diff
@@ -14,35 +14,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+KUBE_ROOT=$(cd $(dirname "${BASH_SOURCE}")/../../.. && pwd)
```

Comment:
No pwd please. Really, the convention in *all* files is:

KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../../..